test/common.js

```diff
@@ -21,6 +76,8 @@ if (process.env.TEST_THREAD_ID) {
 }
 exports.tmpDir = path.join(exports.testDir, exports.tmpDirName);
 
+refreshTmpDir();
```

Comment:
actually - that comment probably isn't fair without actually benchmarking this, so I'll reserve judgement until someone does